src/planwise/client/projects2/components/settings.cljs

```diff
@@ -67,53 +68,51 @@
-                                          :value     (:dataset-id @current-project)
-                                          :on-change #(dispatch [:projects2/save-key :dataset-id %])}]
-            [current-project-input "Capacity workload" [:config :sites :capacity] valid-input]
-            [m/TextFieldHelperText {:persistent true} (str "How many " (get-in @current-project [:config :demographics :unit-name]) " can be handled per site capacity")]]
-
-           [:section {:class-name "project-settings-section"}
-            [section-header 4 "Coverage"]
-            [coverage-algorithm-filter-options {:coverage-algorithm (:coverage-algorithm @current-project)
-                                                :value              (get-in @current-project [:config :coverage :filter-options])
-                                                :on-change          #(dispatch [:projects2/save-key [:config :coverage :filter-options] %])
-                                                :empty              [:div {:class-name " no-dataset-selected"} "First choose dataset."]}]]
-           [:section {:class-name "project-settings-section"}
-            [section-header 5 "Actions"]
-            [current-project-input "Budget" [:config :actions :budget] valid-input]]]]]
-
-        [:div {:class-name "project-settings-actions"}
-         [project-delete-button delete?]
-         [project-start-button {} @current-project]]]
-
-       [new-dialog {:open?     @delete?
-                    :title     "Delete Project"
-                    :accept-fn #(dispatch [:projects2/delete-project (:id @current-project)])
-                    :cancel-fn #(reset! delete? false)
-                    :content   [:p "Do you want to delete this project?"]}]])))
+             [:section {:class-name "project-settings-section"}
+              [section-header 4 "Coverage"]
+              [coverage-algorithm-filter-options {:coverage-algorithm (:coverage-algorithm @current-project)
+                                                  :value              (get-in @current-project [:config :coverage :filter-options])
+                                                  :on-change          #(dispatch [:projects2/save-key [:config :coverage :filter-options] %])
+                                                  :empty              [:div {:class-name " no-dataset-selected"} "First choose dataset."]}]]
+             [:section {:class-name "project-settings-section"}
+              [section-header 5 "Actions"]
+              [current-project-input "Budget" [:config :actions :budget] valid-input]]]]]
+          [:div {:class-name "project-settings-actions"}
+           [project-delete-button delete?]
+           [project-start-button {} @current-project]]]
+         [delete-project-dialog {:open? @delete?
+                                 :cancel-fn hide-dialog
+                                 :delete-fn (fn []
+                                             ;(hide-dialog)
```

Comment:
Is this a leftover? Are you sure we don't need to explicitly hide the dialog here?

Reply:
I should have commented that when the accept dialog button is not being used (in this current version we just use delete), it is also not necessary to use hide-dialog